app/ui/script.js

```diff
@@ -70,13 +70,15 @@ currentWindow.on('history-buttons-change', updateButtons)
 currentWindow.on('page-title-updated', (title) => {
   pageTitle.innerText = title + ' - Agregore Browser'
 })
-
 currentWindow.on('enter-html-full-screen', () => {
   if (!rawFrame) nav.classList.toggle('hidden', true)
 })
 currentWindow.on('leave-html-full-screen', () => {
   if (!rawFrame) nav.classList.toggle('hidden', false)
 })
+currentWindow.on('update-target-url', async (url) => {
```

Comment:
Would it be possible to have a `<span>` that gets displayed over top of the URL field instead so that it can be hidden after we navigate off of a link without having to mess with the navigation?

Reply:
Sure, I've given it a shot!